pkg/apis/provisioning/v0alpha1/jobs.go

```diff
@@ -64,16 +64,16 @@ type JobSpec struct {
 	Repository string `json:"repository"`
 
 	// Pull request options
-	PullRequest *PullRequestOptions `json:"pr,omitempty"`
+	PullRequest *PullRequestJobOptions `json:"pr,omitempty"`
 
 	// Required when the action is `export`
-	Export *ExportOptions `json:"export,omitempty"`
+	Export *ExportJobOptions `json:"export,omitempty"`
 
 	// Required when the action is `sync`
-	Sync *SyncOptions `json:"sync,omitempty"`
+	Sync *SyncJobOptions `json:"sync,omitempty"`
```

Comment:
to avoid confusion with the *repository* settings, this adds "Job" to all the options